### PR TITLE
Ccg trad 2 salt #22376 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

Parts are copied from `modules/client-configuration/pages/registration-bootstrap.adoc` and adjusted.

https://github.com/SUSE/spacewalk/issues/22376

# Target branches

Which documentation version does this PR apply to?
- [x] master (Default)
- [ ] manager-4.3

# Links
- Related development PR #<insert PR link, if any>
- Fixes #<insert spacewalk issue, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
Note, in its current incarnation, we probably can backport this PR to 4.3 painlessly.  Later, on 5.0, we must check whether the default bootstrap script (template) has changes with the variable names, etc.  Additionally, we must check the Web UI for editing the bootstrap script.

File: modules/client-configuration/pages/contact-methods-migrate-traditional.adoc [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+====
+
+
+
+== Run bootstrap script
+
+As the final step, you run the bootstrap script to migrate and register clients.
+
+
+.Procedure: Running the Bootstrap Script
```
Comment (0rnela) [inline modules/client-configuration/pages/contact-methods-migrate-traditional.adoc:84]:
```suggestion
.Procedure: Running the bootstrap script
```

**Review by 0rnela — CHANGES_REQUESTED** [review]

**Review by 0rnela — APPROVED** [review]

File: .changelog [hunk 2]
```diff
@@ -4,12 +4,14 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 
+- Documented migrating from traditional to Salt client in Client
+  Configuration Guide
+- Update Hardware Requirements section about disk space for
+  /var/spacewalk in Installation and Upgrade Guide
```
Comment (keichwa) [inline .changelog:10]:
```suggestion
```

File: .changelog [hunk 3]
```diff
@@ -4,12 +4,14 @@
...
 - Added comment about SCC subscription to Administration Guide
   (bsc#1211270)
 - Fixed link loop and clarified some server upgrade description details
   in the Installation and Upgrade Guide (bsc#1214471)
-- Update Hardware Requirements section about disk space for /var/spacewalk in
-  Installation and Upgrade Guide
```
Comment (keichwa) [inline .changelog:12]:
```suggestion
- Update Hardware Requirements section about disk space for
  /var/spacewalk in Installation and Upgrade Guide
```

File: modules/client-configuration/pages/contact-methods-migrate-traditional.adoc [hunk 4]
```diff
@@ -0,0 +1,133 @@
+[[contact-methods-migrate-trad]]
+= Migrate traditional clients to {salt} clients
+
+To migrate a system from a traditional client to a {salt}, the use can create the {salt} bootstrap script and re-register the client system with it.
```
Comment (admd) [inline modules/client-configuration/pages/contact-methods-migrate-traditional.adoc:4]:
```suggestion
To migrate a system from a traditional client to a {salt}, user can create the {salt} bootstrap script and re-register the client system with it.
```

**Review by admd — APPROVED** [review]
This looks good Karl. Thank you.

**Pull request merged**